pytest_fixtures/sys_fixtures.py

```diff
@@ -21,3 +23,11 @@ def relax_bfa(default_sat):
     """Relax BFA protection against failed login attempts"""
     if default_sat:
         default_sat.cli.Settings.set({'name': 'failed_login_attempts_limit', 'value': '0'})
+
+
+@pytest.fixture(autouse=True, scope='session')
+def proxy_port_range(default_sat):
+    """Assigns port range for fake_capsules"""
+    port_pool_range = settings.fake_capsules.port_range
```

Comment:
The problem that you're running into with unit tests is that `default_sat` is not actually provisioned, there's no target to run on.

That's why the other session scoped autouse fixtures first check `if default_sat` before executing anything.

Reply:
Thanks Mike